### ISSUE TITLE
Navigate menu with arrow keys, auto-select first item

### ISSUE THREAD

**Issue by marisademeglio** [daisy/pipeline-ui#187]
From recent tester feedback:

"when any menu is open pressing left or right arrow key should open next menu item and first item of menu item should be selected"

See also #186

**Comment by marisademeglio** [comment]
@NPavie can you check on windows for the left/right arrow key behavior? It's ok on mac, arrowing left and right moves between the main menus. It does not select the first item but that is consistent with other mac apps.

**Comment by NPavie** [comment]
Same as for #186 : on Windows, chrome-based app don't provide this behavior, but other windows application are (tested visual studio community, Word and vscode).

In that particular case, edge only have a single menu available so it cannot be tested with, but vscode does have the behavior mentionned working with arrow keys for navigating between menu in their menu bar.

**Comment by NPavie** [comment]
Opened an issue regarding this in https://github.com/electron/electron/issues/41324

**Comment by marisademeglio** [comment]
Opened an issue for this: https://github.com/electron/electron/issues/52055

The previous issue had been closed as stale. The behavior still exists though.
